### PR TITLE
Lazy select

### PR DESCRIPTION
**Pull request by nobuti**
This PR fixes https://github.com/CartoDB/support/issues/952.

A new lazy select component to manage the 40 rows per page request's limitation has been created. Basically, when the user fills the search input, a new request is made to grab all items that match with the value filled.

lazy-select

In order to make the acceptance, create a filter analysis with a string column:
- [ ] Check that the select works properly. It mainly covers the same functionality that a [select.
- [ ] Check that the searching works properly.
- [ ] No request returns more than 40 rows.
- [ ] it's important to check if the analysis itself works properly with this change. 

cc @noguerol @ramiroaznar

### CODE REVIEW

**Comment by nobuti** [comment]
could you please take a CR @rubenmoya?

File: lib/assets/core/javascripts/cartodb3/components/form-components/editors/lazy-select/lazy-list-view.js [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  },
+
+  _createListView: function () {
+    if (this._listView) {
+      this._listView.clean();
+      this.removeView(this._listView);
+    }
+
+    this._listView = new CustomListView({
+      className: 'DO-Measurements',
```
[the remark this answers is not in the document]
Comment (nobuti) [reply lib/assets/core/javascripts/cartodb3/components/form-components/editors/lazy-select/lazy-list-view.js]: The styles are the same, but it's true we need a more agnostic name for those. I'll change it.

**Comment by nobuti** [comment]
Probably yes, I'll take a look at a possible refactor.

**Comment by noguerol** [comment]
Anywhere in staging we can test it?

**Comment by nobuti** [comment]
In ded11.

**Comment by nobuti** [comment]
@rubenmoya even the code seems pretty similar, I found subtle differences, especially when working with the form model, that they will do that making a refactor turns into a really hard task. Also, the DO piece of code is kind of special, that's why the functionality we have here in one component, it's there in 3. Because of this, I think we should leave the refactor out of scope for now.

**Comment by rubenmoya** [comment]
Then is better to keep it 3 similar separated components than in a complicated and hard to work with one.

**Comment by ivanmalagon** [comment]
Found a glitch in ded11:

image

**Pull request merged**